lib/fog/aws/requests/storage/get_bucket_location.rb

```diff
@@ -22,8 +22,7 @@ def get_bucket_location(bucket_name)
             :idempotent => true,
             :method   => 'GET',
             :parser   => Fog::Parsers::AWS::Storage::GetBucketLocation.new,
-            :query    => {'location' => nil},
-            :path_style => true
```

Comment:
I'm slightly concerned that this will break something, though only because I imagine we set this for a reason at some point in the past (I wasn't readily able to ascertain the history, due to the extraction from fog/fog).